### PR TITLE
Path to source directory and SampleID included in final report. Shoul…

### PR DESCRIPTION
**Pull request by Blosberg**
…d be sufficient for the user to infer the file.
Also removed a duplicate output of the MethylRaw file.

### CODE REVIEW

**Comment by alexg9010** [comment]
The second output of the methyl raw is in a different section of the report. In the methCall part it was an output, but in the segmentation part it is actually an input (with your change there will be no input to segmentation specified).

**Comment by Blosberg** [comment]
The tests I just ran completed without error. InputParams is defined exactly the same way above. The difference that results from this remove is that 
this table is no longer being rendered for the 2nd time in the same report.  I left in the methyl call section but not in the segmentation section because methyl calls is higher in the hierarchy.

**Comment by alexg9010** [comment]
As I said before "In the methCall part it was an output, but in the segmentation part it is actually an input.(with your change there will be no input to segmentation specified)"

File: report_templates/index.Rmd.in [hunk 1]
```diff
@@ -185,16 +194,6 @@ cat('### Parameters\n')
 cat('**Input parameters:**\n')
 ```
 
-```{r Segmentation.print_params_in2, eval=Segmentation}
```
Comment (alexg9010) [inline report_templates/index.Rmd.in]: So where do you now specify the input to the segmentation.

**Pull request merged**